cypress/integration/ErrorHandling/TwoConsecutiveUserInput.spec.js

```diff
@@ -26,11 +26,11 @@ describe('ErrorHandling', () => {
     editDialogModal.ClickSaveCloseButton()
 
     modelPage.VerifyErrorIconForTrainDialogs()
-    train.VerifyErrorsFoundInTraining(`${common.validityIcon}Hey`, 'world peace', "Sorry $name, I can't help you get $want")
+    train.VerifyErrorsFoundInTraining(`Hey`, 'world peace', "Sorry $name, I can't help you get $want")
```

Comment:
Unrelated to review, but eventually find a way to share these as constant test data object.

Reply:
That is what I thought I did, but when we meet up on Wednesday you can explain to me how to do it the way you like it. I did see how you defined the constants in the "Description and Tag" test case, perhaps that is what you are referring to...and I am not currently following that model, but I can, I have no objection to that.